cmd/snap/cmd_recovery.go

```diff
@@ -43,7 +47,11 @@ func init() {
 	addCommand("recovery", shortRecoveryHelp, longRecoveryHelp, func() flags.Commander {
 		// XXX: if we want more/nicer details we can add `snap recovery <system>` later
 		return &cmdRecovery{}
-	}, nil, nil)
+	}, colorDescs.also(
+		map[string]string{
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"show-recovery-keys": i18n.G("Show recovery keys (if available) to unlock an encrypted partitions."),
```

Comment:
this maybe can be just "--show-keys" ?

unlock *the* encrypted partitions.

The main help should contain something like:

```
--show-keys displays recovery keys that can be used to unlock the encrpyted partitions if device-bound auto-unlock does not work.
```

Reply:
note that you have a typo in the suggestion if that was copy/pasted into @mvo5's branch locally it should be fixed
`encrpyted` -> `encrypted`

also, imho `device-bound` is a little awkward, maybe something like

```
--show-keys displays recovery keys that can be used to unlock the encrypted partitions if the device-specific automatic unlocking does not work.
```

but maybe that's too long and verbose

Reply:
I like this tweak, added.